server/src/services/MailerService.ts

```diff
@@ -86,7 +63,51 @@ export default class MailerService {
     });
   }
 
-  public async sendEmail(): Promise<SentMessageInfo> {
+  public loadValues(data: MailerData): MailerService {
+    this.emailList = data.emailList;
+    this.subject = data.subject;
+    this.htmlEmail = data.htmlEmail;
+    this.backupText = data.backupText || '';
+    this.iCalEvent = data.iCalEvent;
+
+    // to be replaced with env vars
+    this.ourEmail = process.env.CHAPTER_EMAIL || 'ourEmail@placeholder.place';
+    this.emailUsername = process.env.EMAIL_USERNAME || 'project.1';
+    this.emailPassword = process.env.EMAIL_PASSWORD || 'secret.1';
+    this.emailService = process.env.EMAIL_SERVICE;
+    this.emailHost = process.env.EMAIL_HOST || 'localhost';
+
+    if (!process.env.SENDGRID_KEY || !process.env.SENDGRID_EMAIL) {
+      this.createTransporter();
+    }
+
+    return this;
+  }
```

Comment:
I don't believe we need this function.  Reason being, we can always instantiate `MailerService` if we want to test it.

Also, since we're passing `MailerData` to `sendEmail`, it's no longer necessary to keep the details of a specific email (`emailList`, `subject` and so on) as properties of the `MailerService`.